internal/server/iigointernal/judiciary_test.go

```diff
@@ -427,7 +427,7 @@ func TestApplySanctions(t *testing.T) {
 				{
 					ClientID:     shared.Team1,
 					SanctionTier: roles.SanctionTier1,
-					TurnsLeft:    sanctionLength,
+					TurnsLeft:    2,
```

Comment:
tests shouldn't break when the defaults in params are changed. you should be injecting these params into your functions.

Reply:
Fair enough, I think the functions are injected with these params, the test was poorly written. Refactored to use injected values.